frontend/packages/erd-core/src/features/gtm/utils/clickLogEvent.ts

```diff
@@ -2,11 +2,19 @@ import { pushToDataLayer } from './pushToDataLayer'
 
 type ClickLogEvent = {
   element: string
+  zoomLevel?: string
+  showMode?: string
 }
 
-export const clickLogEvent = ({ element }: ClickLogEvent) => {
+export const clickLogEvent = ({
+  element,
+  zoomLevel,
+  showMode,
+}: ClickLogEvent) => {
   pushToDataLayer({
     event: 'click',
     element,
+    zoomLevel,
+    showMode,
```

Comment:
Regarding the toolbar operations, since the design was to record the operations in a log called "toolbarAction", I would like you to create a separate function such as `toolbarActionLogEvent()`. 🙏

Reply:
I apologize for the mistake😱.
 I've revised the code, so please check it again🙏.
https://github.com/liam-hq/liam/pull/326/commits/582ac0e5fcacbfcec7a1496cc726248c63ba5997